sphinx/source/docs/user_guide/examples/js_events.py

```diff
@@ -1,5 +1,5 @@
-""" Demonstration of how to register event callbacks using an adaptation
-of the color_scatter example from the Bokeh gallery
+""" Demonstration of how to register event callbacks using an adaptation of the color_scatter example from the Bokeh gallery
```

Comment:
This just had to be changed because the line break, and then lack of blank final line, was breaking the docstring parsing.

Reply:
I think we should look for solutions. We have a long line-length for code, but generally do try to keep docstrings specifically < 80 chars line length where possible. How is this breaking exactly? Sphinx won't normally care about split lines, is the leading whitespace the issue?

Reply:
I actually think it was the lack of blank line before the final triple-quote. It's been a while since I fixed this, but I remember the error complaining about expecting a blank line and not finding one. We can certainly test as we write all the actual docstrings.

Reply:
I'll check it out locally and see if anything jumps out.